### ISSUE TITLE
Callouts don't work or render if they part of a list item

### ISSUE THREAD

**Issue by amimas** [favoloso/remarkable-admonitions#10]
I am using this in my docusaurus based site (v1). This works really great but I noticed it doesn't work very well inside list item. For example, my markdown could be:

```markdown
- list item 1

    Brief description of this list item

    :::tip
    Here's a tip in this case
    :::

- list item 2
```

The above mardown snippet does get rendered properly except the tip.

If you need more details, please let me know. Thanks.

**Comment by leonardfactory** [comment]
Thank you for reporting, I'll check it soon and I'll update you

**Comment by leonardfactory** [comment]
Hi, this has been solved thanks to @mterrel! Closing this since this is already available on npm (v0.2.2)
